client/js/components/user/DpUserList/DpUserFormFields.vue

```diff
@@ -107,7 +107,7 @@
       <dp-multiselect
         :id="userId + ':userRoles'"
         ref="rolesDropdown"
-        class="u-mb-0_5"
+        class="u-mb-0_5 whitespace-normal"
```

Comment:
shouldn't we fix that within demosplan-ui?

Reply:
`whitespace: normal` is the default. We only need it in this component, because the outer div has `class="whitespace-nowrap"`.